File: cs50x/Week1/mario_hard/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask for height(1-8)
    int height = get_int("What is the height: ");
    while (height < 1)
    {
        height = get_int("What is the height: ");
    }
    while (height > 8)
    {
        height = get_int("What is the height: ");
    }

    // print left-aligned accord. height
    for (int i = 0; i < height; i++)
    {
        int k = 1;
        while (k < (height - i))
        {
            printf(" ");
            k++;
        }
        for (int j = (i + 1); j > 0; j--)
        {
            printf("#");
        }
        printf("  ");
        for (int y = (i + 1); y > 0; y--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week6/sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    height = 9
    while height < 1 or height > 8:
        height = get_int("Height: ")

    for n in range(height):
        for i in range(height - n - 1):
            print(' ', end='')
        print('#' * (n + 1), end='')
        print()


main()
